routes/web.php

```diff
@@ -17,7 +17,11 @@
     // Login backend
     Route::Auth();
 
+    // Dashboard
     Route::get('dashboard', function () {
         return view('backend.dashboard.index');
     })->name('dashboard');
+
+    // User
+        Route::resource('user', 'UserController');
```

Comment:
pls use Route::resource('user','CategoryController', ['as' => 'admin']);
 to add the admin prefix to the route name. Then the route name will be admin.user.index instead of user.index